docs/src/conventions.md

```diff
@@ -106,3 +106,5 @@ based on the following rules.
 - `wrap_array` should be used as default option. `wrap_array_native` should only
   be used when necessary, e.g., to avoid additional overhead when interfacing
   with external C libraries such as HDF5, MPI, or visualization.
+
+# Numerical Types and Stability
```

Comment:
It looks like something went wrong and the docuemntation is missing?

Reply:
No, I am waiting for both of you to make a final decision on how to address integer issues so that I can start the documentation. Now it is just a title. Michael seems to prefer converting each integer to the floating-point type when both options work. Please check the main PR.

Reply:
I see. I was just confused since you requested my review on this PR.

Reply:
Okay. I will upload an initial version. Please wait